src/core/index.ts

```diff
@@ -1,10 +1,54 @@
+/**
```

Comment:
this part is now in a separate file (core-module.ts): https://github.com/SebastianM/angular2-google-maps/blob/master/src/core/core-module.ts#L1-L17

It may help when you compare your changes with the current index.ts from master: https://github.com/SebastianM/angular2-google-maps/blob/master/src/core/index.ts

Probably you have to change nothing in the index.ts but in the core-module.ts